Lib/fontTools/fontBuilder.py

```diff
@@ -263,17 +263,19 @@ def drawTestGlyph(pen):
 # to insert in setupNameTable doc string:
 # print("\n".join(("%s (nameID %s)" % (k, v)) for k, v in sorted(_nameIDs.items(), key=lambda x: x[1])))
 
-_panoseDefaults = dict(
-    bFamilyType=0,
-    bSerifStyle=0,
-    bWeight=0,
-    bProportion=0,
-    bContrast=0,
-    bStrokeVariation=0,
-    bArmStyle=0,
-    bLetterForm=0,
-    bMidline=0,
-    bXHeight=0,
+_panoseDefaults = SimpleNamespace(
```

Comment:
You could use `SimpleNamespace` instead of `dict` directly, there's no needed for the nested `**dict(...)`

Reply:
Thanks for the pointer, I updated the commit